test/Chainweb/Test/Pact/PactInProcApi.hs

```diff
@@ -412,6 +413,179 @@ chainweb213Test bdb mpRefIO pact = do
         ]
     runCut' = runCut testVersion bdb pact (offsetBlockTime second) zeroNoncer noMiner
 
+pact43UpgradeTest :: TestBlockDb -> IO (IORef MemPoolAccess) -> WebPactExecutionService -> IO ()
+pact43UpgradeTest bdb mpRefIO pact = do
+
+  -- run past genesis, upgrades
+  forM_ [(1::Int)..27] $ \_i -> runCut'
+
+  -- run block 28, pre fork
+  setOneShotMempool mpRefIO getBlock1
```

Comment:
```suggestion
  setOneShotMempool mpRefIO getBlock28
```
(name block mempool after the block height, easier to understand below)

Also, these txs in block 28,29,30 can all be in one block/mempool.

Reply:
Fixed.